apps/wallet/src/ui/app/components/accounts/ProtectAccountForm.tsx

```diff
@@ -17,8 +19,12 @@ const formSchema = z.object({
 	confirmedPassword: z.string().nonempty('Required'),
 	acceptedTos: z.literal<boolean>(true),
 	enabledAutolock: z.boolean(),
+	autoLockTimer: z.string(),
```

Comment:
I think this should be a number and we might also need to add some validation (we don't want negative numbers allowed or 0 I imagine)